packages/safe-tools-client/app/services/scheduled-payment-sdk.ts

```diff
@@ -100,12 +99,12 @@ export default class SchedulePaymentSDKService extends Service {
     );
   }
 
-  @action
-  async getScheduledPaymentGasEstimation(
+  @task({ enqueue: true })
```

Comment:
Previous estimation calls should be canceled, right? Would `restartable` be more accurate here?